src/main/java/nextstep/member/Member.java

```diff
@@ -45,7 +47,15 @@ public String getPhone() {
         return phone;
     }
 
+    public MemberRole getRole() {
+        return role;
+    }
+
     public boolean checkWrongPassword(String password) {
         return !this.password.equals(password);
     }
+
+    public boolean hasAdminRole() {
+        return MemberRole.ADMIN.equals(role);
```

Comment:
`MemberRole.ADMIN` 값을 `Member` 클래스가 알 필요 없이
role 도 하나의 enum 객체이니까 role 객체에게 직접 물어보는 형태는 어떨까요~?
```suggestion
        return role != null && role.isAdmin();
```

Reply:
enum 안에 메서드를 정의해보도록 하겠습니다!